pom.xml

```diff
@@ -40,7 +40,7 @@
   </prerequisites>
 
   <properties>
-    <apache-camel.features.tesb.version>2.24.2.tesb17</apache-camel.features.tesb.version>
+    <apache-camel.features.tesb.version>2.24.2.20220211</apache-camel.features.tesb.version>
```

Comment:
For 7.3 target, suffix should stay as `.tesbX`

Reply:
updated.